pro/src/ui-kit/form/__tests-utils__/index.ts

```diff
@@ -1,2 +1,3 @@
-export { queryField, QueryFieldResponse } from './queryField'
+export type QueryFieldResponse<T> = import('./queryField').QueryFieldResponse<T>;
```

Comment:
ca marche pas `export {QueryFieldResponse} from './queryField'` ?

Reply:
nan

Reply:
```suggestion
export type {QueryFieldResponse} from './queryField'
```

ca fait l'affaire :)

Reply:
tu arrives à lui passer le générique ? moi non

Reply:
si c'est bon ça marche, pourtant j'était sur de l'avoir testé...

Reply:
done